src/plugins/data_source/opensearch_dashboards.json

```diff
@@ -6,5 +6,5 @@
   "ui": true,
   "requiredPlugins": ["opensearchDashboardsUtils"],
   "optionalPlugins": [],
-  "extraPublicDirs": ["common/data_sources", "common/util"]
+  "extraPublicDirs": ["common","common/data_sources", "common/util"]
```

Comment:
Seems like a new pattern to me, so it will bypass the lint check?

Reply:
I suppose would, I checked some other usage before modified.
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/data/opensearch_dashboards.json
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/expressions/opensearch_dashboards.json